Homework_2.sql

```diff
@@ -0,0 +1,33 @@
+USE baseball;
```

Comment:
Please remove the old sql file from the repository

Reply:
removed old file.